packages/@sanity/block-tools/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "@sanity/schema": "2.10.0",
+    "@types/jsdom": "^16.2.11",
```

Comment:
This should probably match the version of jsdom actually used?

Reply:
It should. I trusted VS code's suggestion of installing it for me.